schema/pdfpc.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://pdfpc.github.io/schema/pdfpc/2",
+  "description": "Metadata about a presentation used by pdfpc",
+  "title": "pdfpc",
+  "type": "object",
+  "properties": {
+    "pdfpcFormat": {
+      "description": "Version number of this format, currently 2",
+      "type": "integer",
+      "const": 2
+    },
+    "duration": {
+      "description": "Duration of the presentation, in minutes",
+      "type": "integer",
+      "minimum": 1
+    },
+    "startTime": {
+      "description": "Intended start time, in the HH:MM format",
+      "type": "string",
+      "pattern": "^([0-1]?[0-9]|2[0-3]):[0-5][0-9]$"
+    },
+    "endTime": {
+      "description": "Intended end time, in the HH:MM format",
+      "type": "string",
+      "pattern": "^([0-1]?[0-9]|2[0-3]):[0-5][0-9]$"
+    },
+    "endSlide": {
+      "description": "User-defined last slide, 0-based",
+      "type": "integer",
+      "minimum": 0
+    },
+    "savedSlide": {
+      "description": "Bookmarked slide, 0-based",
+      "type": "integer",
+      "minimum": 0
+    },
+    "lastMinutes": {
+      "description": "Number of last minutes when the timer color changes",
+      "type": "integer",
+      "minimum": 1,
+      "default": 5
+    },
+    "disableMarkdown": {
+      "description": "Interpret notes as plain text",
+      "type": "boolean",
+      "default": false
+    },
+    "noteFontSize": {
+      "description": "Font size used to display slide notes",
+      "type": "integer",
+      "minimum": 1,
+      "default": 20
+    },
+    "defaultTransition": {
+      "description": "Default slide transition",
+      "type": "string",
+      "$comment": "A proper regex pattern would be kilometer long...",
+      "default": "replace",
+      "examples": [
```

Comment:
I would leave examples within the man page. Otherwise you have to maintain these informations at two places.

Reply:
But the man page is not versioned... The examples might change from one format version to another.

Reply:
No but the man page corresponds to the installed pdfpc version which is bound to a specific schema version

Reply:
This again relates to whether the schema would be downloadable over the Internet. The local copy is always available in the tarball, of course. We can even `make install` it.

Reply:
I think I found my misunderstanding. Putting examples in the schema is a valid choice then.